run_tests.sh

```diff
@@ -7,6 +7,9 @@ export TF_CPP_MIN_LOG_LEVEL="3"
 # --------------------------------------------------------------------------------------------------------------- TESTS
 
 
+pytest -q -s tests/attacks/evasion/test_brendel_and_bethge.py --mlFramework="pytorch" --durations=0
```

Comment:
1) line 10 should be placed within the main loop starting at line 37. and should use `--mlFramework=$mlFramework` to make sure it is ran with all the frameworks supported by ART. If some of the tests currently do not support a framework, simply notify travis by adding a `@pytest.mark.skipMlFramework("kerastf", "tensorflow", etc.. "non_dl_frameworks")`.

2) Also the `--skip_travis=True` argument should be added to make sure that any tests with a `skip_travis` marker are skipped while running in travis